web/client/components/data/query/CrossLayerFilter.jsx

```diff
@@ -47,6 +48,16 @@ export default ({
         index: 0
     }]} = queryCollection;
 
+    const UnMatchingLayerOptions = layers
```

Comment:
normal variables should have lower camel case. If === 0 no layers are present. also in that case, the popover should show, isn't it?

Reply:
Okay, will change the variable call. I will look at using the current layer to filter